### PR TITLE
[Selenium] Create "FindPreferencesConsoleTab" selenium page object

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Create "FindPreferencesConsoleTab" selenium page object
ddd

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/11201

### CODE REVIEW

**Review by SkorikSergey — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  public void clickOnNode(String visibleText) {
+    for (WebElement node : getNodes()) {
+      if (isNodeContainsText(node, visibleText)) {
+        seleniumWebDriverHelper.waitAndClick(node);
+      }
+    }
+
+    String errorMessage =
+        format("Any node with specified visible text: \"%s\" has not been detected", visibleText);
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java]: `Any node with specified visible text: \"%s\" has not been detected` > `No node with text \"%s\" has been detected`

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  private final SeleniumWebDriverHelper seleniumWebDriverHelper;
+
+  @Inject
+  private FindPreferencesConsoleTab(SeleniumWebDriverHelper seleniumWebDriverHelper) {
+    this.seleniumWebDriverHelper = seleniumWebDriverHelper;
+  }
+
+  public interface Locators {
+    String FIND_PREFERENCES_CONSOLE_BUTTON_ID = "gwt-debug-partButton-Find References";
+    String ALL_FIND_REFERENCES_CONSOLE_NODES_XPATH =
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java]: `REFERENCES` or `PREFERENCES` ?

**Comment by dmytro-ndp** [comment]
Looks like it is about "Find References Console Tab". But it is confused with `preferences` in the class.

**Comment by dmytro-ndp** [comment]
Also IMHO it would be more clear if we use `reference` term instead of `node` inside the class.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        "//div[@id='gwt-debug-LS-open-location-panel']//div[@id='content-Tree']/div";
+    String SELECTED_NODE_XPATH =
+        ALL_FIND_REFERENCES_CONSOLE_NODES_XPATH + "//div[contains(@class, 'selected')]";
+  }
+
+  public void clickOnPreferencesConsoleButton() {
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java]: `clickOnFindReferencesConsoleButton` ?

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java (hunk 2 above)
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindPreferencesConsoleTab.java]: FOUND_REFERENCES_LIST_XPATH`?

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**